AutomaticComponentToolkit/lib3mf.xml

```diff
@@ -1023,682 +1787,2491 @@
+			<param name="Identifier" type="string" pass="in"
+				description="the identifier of the node" />
+			<param name="Configuration" pass="in" type="enum" class="ImplicitNodeConfiguration"
+				description="the configuration of the node" />
+			<param name="DisplayName" pass="in" type="string"
+				description="the display name of the node" />
+			<param name="Tag" pass="in" type="string"
+				description="the tag of the node" />
+			<param name="Node" pass="return" type="class" class="LogNode"
+				description="the added node" />
+		</method>
+
+		<method name="AddLog2Node" description="Add a Log2Node">
+			<param name="Identifier" pass="in" type="string"
+				description="the identifier of the node" />
+			<param name="Configuration" pass="in" type="enum" class="ImplicitNodeConfiguration"
+				description="the configuration of the node" />
+			<param name="DisplayName" pass="in" type="string"
+				description="the display name of the node" />
+			<param name="Tag" pass="in" type="string"
+				description="the tag of the node" />
+			<param name="Node" pass="return" type="class" class="Log2Node"
+				description="the added node" />
+		</method>
+
+		<method name="AddLog10Node" description="Add a Log10Node">
+			<param name="Identifier" pass="in" type="string"
+				description="the identifier of the node" />
+			<param name="Configuration" pass="in" type="enum" class="ImplicitNodeConfiguration"
+				description="the configuration of the node" />
+			<param name="DisplayName" pass="in" type="string"
+				description="the display name of the node" />
+			<param name="Tag" pass="in" type="string"
+				description="the tag of the node" />
+			<param name="Node" pass="return" type="class" class="Log10Node"
+				description="the added node" />
+		</method>
+
+		<method name="AddLengthNode" description="Add a LengthNode">
+			<param name="Identifier" pass="in" type="string"
+				description="the identifier of the node" />
+			<param name="DisplayName" pass="in" type="string"
+				description="the display name of the node" />
+			<param name="Tag" pass="in" type="string"
+				description="the tag of the node" />
+			<param name="Node" pass="return" type="class" class="LengthNode"
+				description="the added node" />
+		</method>
+
+		<method name="AddTransposeNode" description="Add a TransposeNode">
+			<param name="Identifier" pass="in" type="string"
+				description="the identifier of the node" />
+			<param name="Configuration" pass="in" type="enum" class="ImplicitNodeConfiguration"
+				description="the configuration of the node" />
+			<param name="DisplayName" pass="in" type="string"
+				description="the display name of the node" />
+			<param name="Tag" pass="in" type="string"
+				description="the tag of the node" />
+			<param name="Node" pass="return" type="class" class="TransposeNode"
+				description="the added node" />
+		</method>
+
+		<method name="InverseNode" description="Add a InverseNode">
```

Comment:
Instead of InverseNode, it should be AddInverseNode to follow other method naming convention.

Reply:
agree